packages/lib/versionInfo.test.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
this file shouldn't be here, it's generated by Typescript compiler

Reply:
I will delete that file.

Reply:
I have deleted the .test.js file from commit.

Reply:
you've deleted the TypeScript file instead...

Reply:
Sorry I deleted that by mistake. I'm reverting the changes and will delete the js file.